packages/manager/apps/dedicated/client/app/dedicated/server/terminate/dedicated-server-terminate.controller.js

```diff
@@ -21,10 +23,12 @@ angular.module('App').controller(
       this.Server = Server;
       this.DedicatedServerFeatureAvailability = DedicatedServerFeatureAvailability;
       this.Alerter = Alerter;
+      this.coreConfig = coreConfig;
```

Comment:
`coreConfig` needed even though we have `Feature flipping` ?

Reply:
This is removed. Thanks